src/Document.js

```diff
@@ -187,6 +177,39 @@ export default class Document extends ParentNode {
         return null;
     }
 
+    get documentElement() {
+        return this._documentElement;
+    }
+
+    get head() {
+        let head = this._documentElement.getElementsByTagName('head').shift();
+        if (!head) {
```

Comment:
According to [the w3c spec](https://www.w3.org/TR/html5/dom.html#dom-document-head), the head can return null:

```
The head element of a document is the first head element that is a child of the html element, if there is one, or null otherwise.
```